lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -269,6 +298,24 @@ function dosomething_global_convert_country_to_language($country) {
 }
 
 /**
+ * Converts the given language into its assigned URL prefix.
+ *
+ * @param string language
+ *  The language to get the prefix for (eg: 'en', 'mx')
+ *
+ * @return
+ *  The URL prefix mapped to this language
+ */
+function dosomething_global_get_prefix_for_language($language) {
```

Comment:
Drupal has a function language_list() that returns all of the languages with their information. This query works, but I'm thinking that using the built-in function is more drupally, and it's a little less code for us to maintain.
```
Array
(
    [en] => stdClass Object
        (
            [language] => en
            [name] => English
            [native] => English
            [direction] => 0
            [enabled] => 1
            [plurals] => 0
            [formula] => 
            [domain] => 
            [prefix] => us
            [weight] => 0
            [javascript] => 
        )

    [en-global] => stdClass Object
        (
            [language] => en-global
            [name] => English, Global
            [native] => English, Global
            [direction] => 0
            [enabled] => 1
            [plurals] => 0
            [formula] => 
            [domain] => 
            [prefix] => en-global
            [weight] => 0
            [javascript] => 
        )

    [pt-br] => stdClass Object
        (
            [language] => pt-br
            [name] => Portuguese, Brazil
            [native] => Português Brasileiro
            [direction] => 0
            [enabled] => 1
            [plurals] => 2
            [formula] => ($n!=1)
            [domain] => 
            [prefix] => br
            [weight] => 0
            [javascript] => 
        )

    [es-mx] => stdClass Object
        (
            [language] => es-mx
            [name] => Spanish, Mexico
            [native] => Español Mexicano
            [direction] => 0
            [enabled] => 1
            [plurals] => 0
            [formula] => 
            [domain] => 
            [prefix] => mx
            [weight] => 0
            [javascript] => 
        )

)
```